### PR TITLE
KOGITO-6262: Extend new boxed expression editor decision table coverage

### PR DESCRIPTION
**Pull request by jomarko**
KOGITO-6262: Extend new boxed expression editor decision table coverage
    
For more details see:
- https://issues.redhat.com/browse/KOGITO-6262

Dependent on #159

### CODE REVIEW

**Review by vpellegrino — APPROVED** [review]
Test cases look good to me. Thanks @jomarko

**Review by yesamer — APPROVED** [review]

File: kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts [hunk 1]
```diff
@@ -15,4 +15,129 @@ describe("Context Expression Tests", () => {
...
+    // insert one output column right
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert right").click({ force: true });
+
+    // insert one output column left
+    cy.ouiaType("expression-column-header-cell-info").contains("output-1").rightclick();
+    cy.contains("Insert left").click({ force: true });
+
+    cy.get("th:contains('output-')").should(($outputs) => {
+      expect($outputs).to.have.length(4);
```
Comment (ljmotta) [inline kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts:105]: The length shouldn't be 3? If not, could you please explain? I'm not used to Cypress tests.

**Review by ljmotta — COMMENTED** [review]
Hi @jomarko , I've made a comment inline.

File: kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts (hunk 1 above)
Comment (jomarko) [reply kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts:105]: @ljmotta good question

It is this scenario when a decision table is in the last context entry:
Screenshot from 2021-11-29 16-55-57

```json
{"uid":"id3","logicType":"Context","name":"Expression Name","dataType":"<Undefined>","contextEntries":[{"entryInfo":{"name":"ContextEntry-1","dataType":"<Undefined>"},"entryExpression":{"name":"ContextEntry-1","dataType":"<Undefined>"},"nameAndDataTypeSynchronized":true,"id":"_E2CB8F71-B220-48FE-85D7-27CD11084613"}],"result":{"uid":"id16","logicType":"Decision Table","hitPolicy":"UNIQUE","aggregation":"","input":[{"id":"_1B55A056-BA9A-47A7-A14B-3FA556959E45","name":"input-1","dataType":"<Undefined>"}],"output":[{"id":"_36696549-A53E-4C53-B52B-2D82F509E8FC","name":"column-3","dataType":"<Undefined>","width":100},{"id":"_233021AA-6808-49D3-B3DF-9F4948C89870","name":"output-1","dataType":"<Undefined>"},{"id":"_4A41C245-9526-43D9-8CC0-EFE891912194","name":"column-2","dataType":"<Undefined>","width":100}],"annotations":[{"id":"_5C433657-D070-47B1-BC2C-89135EDA4E27","name":"annotation-1","width":100}],"rules":[{"id":"_44BC8FB6-D41F-4412-82C3-5C890213D972","inputEntries":["-"],"outputEntries":["","",""],"annotationEntries":[""]}]},"entryInfoWidth":150,"entryExpressionWidth":584,"noClearAction":false}
```

running a showcase on main, without #171

Comment (ljmotta) [reply kogito-editors-js/packages/boxed-expression-component/showcase/cypress/integration/context_expression_spec.ts:105]: Oh ok, now I get it.

**Review by ljmotta — APPROVED** [review]

**Comment by jomarko** [comment]
PR rebased to see if CI is fine after #173 merge

**Comment by jomarko** [comment]
@yesamer please merge

**Pull request merged**